src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheckTest.java

```diff
@@ -123,7 +123,7 @@ public void testAnother4()
     }
 
     @Test
-    public void testScopes() throws Exception {
+    public void testScopesOne() throws Exception {
```

Comment:
We need to avoid nonsense test naming conventions like this; if we can't come up with some descriptive name to differentiate these tests, we need to reconsider the content and motivation for them.